sagemaker_batch_transform/pytorch_flores_batch_transform/sagemaker_batch_transform_torchserve.ipynb

```diff
@@ -0,0 +1,361 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    # The job should complete in appxorimately 7~10 minutes, depending on the instance type
</pre><p>Typo - <code>appxorimately</code>  -&gt; <code>approximately</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Done, thanks